_activities/cwp.md

```diff
@@ -61,7 +61,8 @@ or as a [PDF]({{site.docs.url}}/CWP/papers/roadmap/HSF-Community-White-Paper-v0.
 
 This is a list of recent talks about the CWP which were given by various HSF members:
 
-* 2017-11-04 [Talk](https://indico.cern.ch/event/658716/contributions/2685752/attachments/1552099/2454959/20171103-ITTF-HSFCWP-AV.pdf) to the [CERN IT Technical Forum](https://indico.cern.ch/event/658716/) by Andrea Valassi.
+* 2017-11-28 [Presentation](https://indico.desy.de/indico/event/18681/session/8/contribution/114/material/slides/0.pdf) at the [11th Terascale Alliance Workshop](https://indico.desy.de/indico/event/18681/) by Benedikt Hegner.
+* 2017-11-04 [Presentation](https://indico.cern.ch/event/658716/contributions/2685752/attachments/1552099/2454959/20171103-ITTF-HSFCWP-AV.pdf) to the [CERN IT Technical Forum](https://indico.cern.ch/event/658716/) by Andrea Valassi.
```

Comment:
Unfortunately this talk is behind SSO, so it will not work for anyone without a CERN ID. I don't know why the URL checker doesn't pick this up; perhaps it thinks the SSO redirect is the talk URL!